geomesa-kafka/geomesa-kafka-confluent/src/main/scala/org/locationtech/geomesa/kafka/confluent/ConfluentFeatureSerializer.scala

```diff
@@ -122,5 +100,27 @@ class ConfluentFeatureSerializer(
 
   override def serialize(feature: SimpleFeature, out: OutputStream): Unit =
     throw new NotImplementedError("ConfluentSerializer is read-only")
-}
 
+  private class ConfluentFeatureReader(sft: SimpleFeatureType, schema: Schema) {
```

Comment:
About line 102, now that you've thought about things, @fdfea, do you think it'd be possible to make the Confluent KDS read-write?

Reply:
So I would need to translate SFT to Schema? I think I could implement the serialize method, but I don't quite get how the whole thing would actually work. You write simple features to the store and then you can read the avro from a kafka topic?

Reply:
That should probably be another ticket

Reply:
yeah i don't think writing is necessary at this point, although that would make it easier to integrate with schema registry.

Reply:
Yeah, the GM CKDS FeatureStore would write SimpleFeatures to the topic as Avro.  And yeah, it could/should be a new ticket.